charts/vector/templates/podmonitor.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: PodMonitor
 metadata:
   name: {{ include "vector.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Stumbled across this as I was searching through PRs to figure out the review process. This is going to "conflict" with the PR I just created:

https://github.com/vectordotdev/helm-charts/pull/464